### ISSUE TITLE
autocomplete attribute, and its values

### ISSUE THREAD

**Issue by ffoodd** [ffoodd/a11y.css#430]
`autocomplete` inputs when possible is now required in [WCAG 2.1 1.3.1 SC](https://www.w3.org/TR/WCAG21/#identify-input-purpose) and [RGAA 4.1 11.13 criteria](https://www.numerique.gouv.fr/publications/rgaa-accessibilite/methode-rgaa/criteres/#crit-11-13), with [a predefined set of values](https://www.numerique.gouv.fr/publications/rgaa-accessibilite/methode-rgaa/glossaire/#liste-des-valeurs-possibles-pour-l-attribut-autocomplete) from HTML specs.

Two checks in my mind:
- an error for invalid attribute values,
- a warning for inputs without `[autocomplete]`.

**Comment by PhilippeVay** [comment]
HTML now allows for this https://html.spec.whatwg.org/multipage/forms.html#enabling-client-side-automatic-filling-of-form-controls  
`autocomplete="billing tel"`.

WCAG references HTML LS : https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofilling-form-controls:-the-autocomplete-attribute where some value examples have 3 parts.  

The predefined part is always at the end so `$=` selector should be fine.
